src/components/InventoryTable/EntityTable.js

```diff
@@ -147,7 +147,11 @@ const EntityTable = ({
                     <TableHeader />
                     <TableBody />
                 </PfTable> :
-                <SkeletonTable colSize={ columns.current?.length || 3 } rowSize={ 15 } />
+                <SkeletonTable
+                    colSize={ columns.current?.length || 3 }
+                    rowSize={ 15 }
+                    variant={variant ? variant : tableProps.variant}
```

Comment:
You can use short cutting here
```suggestion
                    variant={variant ?? tableProps.variant}
```